docs/user-guide/migrating-from-jscs.md

```diff
@@ -0,0 +1,79 @@
+# Migrating from JSCS
+
+In April 2016, we [announced](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint) that the JSCS project was shutting down and the JSCS team would be joining the ESLint team. This guide is intended to help those who are using JSCS to migrate their settings and projects to use ESLint. We've tried to automate as much of the conversion as possible, but there are some manual changes that are needed.
+
+## Terminology
```

Comment:
Wouldn't work i think, jstransform output, as far as i can tell, is not based on tokens list but on AST i.e. it will mangle the whole file while changing the comments. It could be a eslint rule which would use polyjuice API with autofixable compatibilities. But this is seems as very unrewarding task, not sure if we could find contributors for that. Seems we are out of luck on this one :/